modules/ROOT/pages/about-administration-tasks.adoc

```diff
@@ -3,13 +3,13 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Exchange provides those with the Exchange Administrator capabilities not shared with Exchange Contributors or Exchange Viewers.
+Anypoint Exchange provides those with the Exchange Administrator role capabilities not shared with Exchange Contributors or Exchange Viewers.
```

Comment:
This sentence is a bit confusing. Should there be a comma after "... Administrator role" ? Also, what's the objective of the sentence. We all know that Admins will have different capabilities than Contribs or viewers. Unless you call out those differences, I don't see a point in this sentence.

Reply:
Good point. I spotted that this sentence was unclear and tried to add a small fix, but I think it's better to remove this one.